### PR TITLE
[DSD-3641] updated properties

### PR DESCRIPTION
**Pull request by Rakshithb1**

### CODE REVIEW

**Review by anshulv1401 — APPROVED** [review]

File: esignet-mock.properties [hunk 1]
```diff
@@ -90,6 +91,7 @@ mosip.esignet.integration.authenticator=MockAuthenticationService
 mosip.esignet.integration.key-binder=MockKeyBindingWrapperService
 mosip.esignet.integration.audit-plugin=LoggerAuditService
 mosip.esignet.integration.captcha-validator=GoogleRecaptchaValidatorService
+mosip.esignet.integration.vci-plugin=IdaVCIssuancePluginImpl
```
Comment (ghost) [inline esignet-mock.properties]: change the value to MockVCIssuancePlugin

File: esignet-default.properties [hunk 2]
```diff
@@ -127,12 +135,15 @@ mosip.esignet.mock.authenticator.kyc-exchange-url=https://${mosip.api.public.hos
...
 mosip.esignet.mock.authenticator.send-otp=https://${mosip.api.public.host}/v1/mock-identity-system/send-otp
 mosip.esignet.mock.supported.bind-auth-factor-types={'WLA'}
+mosip.esignet.mock.vciplugin.verification-method=${mosip.esignet.vci.authn.jwk-set-uri}
 ## ------------------------------------------ oauth & openid supported values ------------------------------------------
 
 ## supported scopes
 mosip.esignet.supported.authorize.scopes={'Manage-Identity-Data','Manage-VID','Manage-Authentication','Manage-Service-Requests','Manage-Credentials'}
 mosip.esignet.supported.openid.scopes={'profile','email','phone'}
 mosip.esignet.openid.scope.claims={'profile' : {'name','address','gender','birthdate','picture','email','phone_number'},'email' : {'email'}, 'phone' : {'phone_number'}}
+mosip.esignet.supported.credential.scopes={'sample_vc_ldp', 'mosip_identity_vc_ldp'}
```
Comment (ghost) [inline esignet-default.properties]: Remove 'sample_vc_ldp' from here

File: esignet-default.properties [hunk 3]
```diff
@@ -127,12 +135,15 @@ mosip.esignet.mock.authenticator.kyc-exchange-url=https://${mosip.api.public.hos
...
+mosip.esignet.credential.scope-resource-mapping={'sample_vc_ldp' : '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', 'mosip_identity_vc_ldp': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential'}
```
Comment (ghost) [inline esignet-default.properties]: Remove 'sample_vc_ldp' mapping from here

File: esignet-default.properties [hunk 4]
```diff
@@ -296,21 +319,91 @@ mosip.kernel.keymanager.jwtsign.validate.json=false
...
+mosip.esignet.vci.authn.filter-urls={ '${server.servlet.path}/vci/credential' }
+mosip.esignet.vci.authn.issuer-uri=${mosip.esignet.domain.url}${server.servlet.path}
+mosip.esignet.vci.authn.jwk-set-uri=${mosip.esignet.domain.url}${server.servlet.path}/oauth/.well-known/jwks.json
+mosip.esignet.vci.authn.allowed-audiences={ '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential' }
+
+mosip.esignet.cnonce-expire-seconds=40
+mosip.esignet.vci.supported.jwt-proof-alg={'RS256'}
+mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}',         \
+  'credential_endpoint': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', \
+  'credentials_supported': {{\
```
Comment (ghost) [inline esignet-default.properties:357]: Lets remove first element from this list

File: esignet-mock.properties [hunk 5]
```diff
@@ -133,7 +141,9 @@ mosip.esignet.mock.authenticator.ida.otp-channels=${mosip.esignet.authenticator.
 ## supported scopes
 mosip.esignet.supported.authorize.scopes={'Manage-Identity-Data','Manage-VID','Manage-Authentication','Manage-Service-Requests','Manage-Credentials'}
 mosip.esignet.supported.openid.scopes={'profile','email','phone'}
-mosip.esignet.openid.scope.claims={'profile' : {'name','given_name','family_name','address','gender','birthdate','picture','email','phone_number','locale'},'email' : {'email'}, 'phone' : {'phone_number'}}
+mosip.esignet.openid.scope.claims={'profile' : {'name','address','gender','birthdate','picture','email','phone_number'},'email' : {'email'}, 'phone' : {'phone_number'}}
+mosip.esignet.supported.credential.scopes={'sample_vc_ldp', 'mosip_identity_vc_ldp'}
```
Comment (ghost) [inline esignet-mock.properties]: Remove 'mosip_identity_vc_ldp'

File: esignet-mock.properties [hunk 6]
```diff
@@ -133,7 +141,9 @@ mosip.esignet.mock.authenticator.ida.otp-channels=${mosip.esignet.authenticator.
...
+mosip.esignet.credential.scope-resource-mapping={'sample_vc_ldp' : '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', 'mosip_identity_vc_ldp': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential'}
```
Comment (ghost) [inline esignet-mock.properties]: Remove 'mosip_identity_vc_ldp' mapping from here

File: esignet-mock.properties [hunk 7]
```diff
@@ -301,16 +322,88 @@ mosip.kernel.keymgr.hsm.health.check.enabled=true
...
+  'name': { 'display': {{'name': 'Given Name', 'locale': 'en' }}}, \
+  'age': { 'display': {{ 'name': 'Age', 'locale': 'en'}}}\
+   }},\
+  'display': {{'name': 'Sample Verifiable Credential by e-Signet', \
+                'locale': 'en', \
+                'logo': {'url': '${mosip.esignet.domain.url}/logo.png',\
+                'alt_text': 'a square logo of a MOSIP'},\
+                'background_color': '#12107c',\
+                'text_color': '#FFFFFF'}}\
+     \ },{\
```
Comment (ghost) [inline esignet-mock.properties]: lets remove the second element from here

File: id-authentication-default.properties [hunk 8]
```diff
@@ -282,7 +286,8 @@ mosip.auth.filter_disable=false
 ida-cache-ttl-in-days=1
 ## To disable cache, set value to NONE, otherwise SIMPLE to enable cache.
 ## Value is based on CacheType enum provided by Spring Boot
-spring.cache.type=SIMPLE
+## spring.cache.type=SIMPLE
+spring.cache.type=NONE
```
Comment (ghost) [inline id-authentication-default.properties:290]: What is the value in qa-triad for this? @Rakshithb1

File: id-authentication-default.properties [hunk 9]
```diff
@@ -417,7 +422,7 @@ kernel.retry.exponential.backoff.max.interval.millisecs=1000
 ## Whether to traverse to the root cause exception from the exception thrown and use the same root cause to decide whether to retry or not. Default is true.
 kernel.retry.traverse.root.cause.enabled=false
 ## Comma separated List of fully qualified Exceptions which are retryable (inclusion list). Their subclasses will also be considered in the evaluation.
-kernel.retry.retryable.exceptions=io.mosip.idrepository.core.exception.IdRepoRetryException,org.springframework.dao.DataIntegrityViolationException,org.hibernate.exception.ConstraintViolationException,org.springframework.orm.ObjectOptimisticLockingFailureException
+kernel.retry.retryable.exceptions=io.mosip.idrepository.core.exception.IdRepoRetryException,org.springframework.dao.DataIntegrityViolationException,org.hibernate.exception.ConstraintViolationException,org.springframework.orm.ObjectOptimisticLockingFailureExceptionf
```
Comment (ghost) [inline id-authentication-default.properties]: Is this valid ? @Rakshithb1  
cc: @mahammedtaheer

File: id-authentication-default.properties (hunk 8 above)
Comment (Rakshithb1) [reply id-authentication-default.properties:290]: NONE

**Pull request merged**